### ISSUE TITLE
Update misleading docs for poetry-install pre-commit hook

### ISSUE THREAD

**Issue by emirkmo** [python-poetry/poetry#9901]
### Issue Kind

Error in existing documentation

### Existing Link

https://python-poetry.org/docs/pre-commit-hooks/#poetry-install

### Description

The poetry-install hook installs poetry and your project dependencies, to a completely isolated system virtual environment managed by pre-commit. This environment is not used for anything else other than this one hook itself. So it is basically a no-op in 99% of cases. The only thing it "tests" is that the currently configured project **can** be installed somewhere. But really poetry itself already does this else how were the lockfiles etc. generated? (It can happen, say with the cache issues we've seen around 1.6)

## Problem

Basically: the hook does not do what the docs say and is extremely misleading. This has been a source of confusion for quite a while: #3255 #9393 

## Suggestion

Reword to say: that this is basically a useless hook that tests that poetry install command works for the current project using a separate global virtual environment managed by pre-commit. But it won't actually install the package for your current environment or be used in your project or with other pre-commit hooks in anyway. 

(Alternate suggestion is to delete the hook)

**Comment by dimbleby** [comment]
duplicate #9393 

if you want to propose docs changes, much the most effective way is to submit a merge request

**Comment by radoering** [comment]
Closing as duplicate of #9393. The docs are only misleading because the hook is buggy so that I think #9393 is sufficient to track this.
